include/baresip.h

```diff
@@ -285,6 +285,7 @@ struct config_sip {
 	char cert[256];         /**< SIP Certificate                */
 	char cafile[256];       /**< SIP CA-file                    */
 	enum sip_transp transp; /**< Default outgoing SIP transport protocol */
+	bool sverify;           /**< Enable SIP TLS verify server   */
```

Comment:
maybe we should align with libre `verify_server`

Reply:
may I squash the commit?